{
    [1, 2].filter(fun() { True; });
}

// args: check
// expected exit status: 1
// expected stdout:
// Error: Expected `Fun<(T), Bool>` argument but got `Fun<(), Bool>`.
// 
// --> src/check_test_files/type_error_arity_lambda.gdn:2
//     [1, 2].filter(fun() { True; });
//                   ^^^^^^^^^^^^^^^

